homeassistant/components/zha/core/helpers.py

```diff
@@ -6,6 +6,7 @@
 """
 import asyncio
 import logging
+from concurrent.futures._base import TimeoutError as Timeout
```

Comment:
Why do we import from this path? Is it different than https://docs.python.org/3/library/concurrent.futures.html#exception-classes?

Reply:
fixed

Reply:
Is it the builtin `TimeoutError`? So it's not the one in concurrent.futures?